Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -126,7 +126,19 @@ private void ensurePowerModeForDrive() {
         ensureModeForTalon(leftDrive, TalonControlMode.PercentVbus);
         ensureModeForTalon(rightDrive, TalonControlMode.PercentVbus);
     }
-
+    
+    public void tankDriveVelocity(double leftVelocity, double rightVelocity){
```

Comment:
you'll need Seth's branch for this to really work so might want to wait for him to merge in first and then update to use the method he added to convert to the correct units here